<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="user" name="Attendance Details Result" tests="2" failures="2" timestamp="1 Aug 2018 08:41:08 GMT" time="0.101" errors="0">
  <testcase name="TC_001_verify_Attendance_DetailsPage" time="0.078" classname="com.cogniphi.MuthootServillance.TestSuite.AttendanceSuite">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:882)
	at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:104)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
	at UTILITY.SafeActions.click_on_the_field(SafeActions.java:52)
	at com.cogniphi.MuthootServillance.AttendanceDetails.open_Attendance_Details_Page(AttendanceDetails.java:40)
	at com.cogniphi.MuthootServillance.TestSuite.AttendanceSuite.TC_001_verify_Attendance_DetailsPage(AttendanceSuite.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_001_verify_Attendance_DetailsPage -->
  <testcase name="TC_0026_status" time="0.002" classname="com.cogniphi.MuthootServillance.TestSuite.AttendanceSuite">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:882)
	at org.openqa.selenium.support.ui.FluentWait.<init>(FluentWait.java:104)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:71)
	at org.openqa.selenium.support.ui.WebDriverWait.<init>(WebDriverWait.java:45)
	at UTILITY.SafeActions.getStatusOfEachVehicle(SafeActions.java:366)
	at com.cogniphi.MuthootServillance.AttendanceDetails.markAttendanceForAllGuard(AttendanceDetails.java:333)
	at com.cogniphi.MuthootServillance.TestSuite.AttendanceSuite.TC_0026_status(AttendanceSuite.java:167)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- TC_0026_status -->
</testsuite> <!-- Attendance Details Result -->
